python/tests/test_options.py

```diff
@@ -71,6 +78,31 @@ def test_accepts_selenium_browser_capabilities_instance(self):
         assert sauce.browserVersion == 'latest'
 
 
+class TestSauceSpecificOptions(object):
+
+    def test_default_build_value(self):
+        sauce = SauceOptions()
+
+        assert sauce.build is not None
+
+    def test_default_test_name(self):
+        sauce = SauceOptions()
+
+        assert sauce.name is not None
+
+    def test_add_test_name(self):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://app.codacy.com/app/SauceLabs/simple_sauce/pullRequest?prid=4626732)